ios/docs/Controls/Card.md

```diff
@@ -0,0 +1,35 @@
+# Card
+
+## Overview
+The `Card` component is a framework for organizing content within the confines of a container. It's main function is to provide the scaffolding for hosting actions and content for a single topic within a card.
```

Comment:
Its* main function

Reply:
Nice catch! I got this from https://react.fluentui.dev/?path=/docs/preview-components-card-card--default. Do you happen to know who owns this page?